proposed/http-middleware/middleware-meta.md

```diff
@@ -123,11 +120,10 @@ Middleware taking this approach generally has the following commonalities:
   the request for processing.
 * The middleware is passed 2 arguments during invocation:
   1. An object that represents an HTTP request.
-  2. A delegate that receives the request to dispatch next middleware in the pipeline.
+  2. A delegate that receives the request to dispatch next middleware.
```

Comment:
> Perhaps this could be: "A delegate by which the middleware may process the request if unable to complete the request alone, or unable to generate a response."

Spot on.

I particularly approve adding the indication that any sort of underlying data structure is an implementation detail, but we must remember that the spec itself still implies that a collection of middlewares will be processed in some sort of sequence, which to be honest will probably be the case in 99.9% of the implementations, but the spec doesn't really prevent from using delegates in some funky ways, and this may be worth an explicit reminder, just to be safe.